File: URI/python/1048.py
salario = float(input())

if salario <= 400:
    percentual = 15 / 100
elif salario <= 800:
    percentual = 12 / 100
elif salario <= 1200:
    percentual = 10 / 100
elif salario <= 2000:
    percentual = 7 / 100
elif salario > 2000:
    percentual = 4 / 100    

reajuste = percentual * salario
novo_salario = salario + reajuste

print('Novo salario: {:.2f}'.format(novo_salario))
print('Reajuste ganho: {:.2f}'.format(reajuste))
print('Em percentual: {} %'.format(int(100 * percentual)))

File: uri/c/1048.c
#include <stdio.h>

int main() {

    float salario;
    float reajuste, novoSalario, reajustePercentual;
    int percentual;

    scanf("%f", &salario);

    if (salario >= 0 && salario <= 400.00) {
        percentual = 15;
    } else if (salario >= 400.01 && salario <= 800.00) {
        percentual = 12;
    } else if (salario >= 800.01 && salario <= 1200.00) {
        percentual = 10;
    } else if (salario >= 1200.01 && salario <= 2000.00) {
        percentual = 7;
    } else if (salario > 2000) {
        percentual = 4;
    }

    reajustePercentual = (float) percentual / 100.0;
    reajuste = (float) (reajustePercentual * salario);
    novoSalario = salario + reajuste;

    printf("Novo salario: %.2f\n", novoSalario);
    printf("Reajuste ganho: %.2f\n", reajuste);
    printf("Em percentual: %d %\n", percentual);

    return 0;

}